insta-ui/src/components/ChatInterface/InputBar.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import { AiOutlinePaperClip, AiOutlineSend } from "react-icons/ai";
```

Comment:
You can take a look at this reference [[ref]](https://github.com/junkaiman/ui-generator/blob/7d9601a94f0d243f0f8319ec434732e9c9150fe4/insta-ui/src/components/SideBar/SideBarItem.tsx#L2C1-L5C1) to replace with font-awesome icon

Reply:
resolved